doc/api/corepack.md

```diff
@@ -96,31 +96,35 @@ it can conflict with such environments. To avoid that happening, call the
 the same time you're preparing your deploy image). This will ensure that the
 required package managers are available even without network access.
 
-The `pack` command has [various flags][]. Consult the detailed
+The `corepack` command has [various flags][]. Consult the detailed
```

Comment:
This is continuing from the previous paragraph so it's referring to `corepack pack`.